tests/sentry/hybrid_cloud/test_organizationmapping.py

```diff
@@ -83,10 +88,72 @@ def test_upsert__update_if_found(self):
         assert fixture_org_mapping.status == OrganizationStatus.PENDING_DELETION
 
     def test_upsert__duplicate_slug(self):
-        self.organization = self.create_organization(slug="alreadytaken")
+        self.organization = self.create_organization(slug="alreadytaken", region="us")
 
         with pytest.raises(IntegrityError):
             organization_mapping_service.upsert(
                 organization_id=7654321,
-                update=RpcOrganizationMappingUpdate(slug=self.organization.slug),
+                update=RpcOrganizationMappingUpdate(slug=self.organization.slug, region_name="us"),
             )
+
+    def test_org_slug_reservation_region_mismatch(self):
+        self.organization = self.create_organization(slug="santry", region="us")
+
+        with pytest.raises(OrganizationMappingConsistencyException):
+            organization_mapping_service.upsert(
+                organization_id=self.organization.id,
+                update=RpcOrganizationMappingUpdate(
+                    slug=self.organization.slug, name="saaaaantry", region_name="eu"
+                ),
+            )
+
+    def test_org_slug_reservation_slug_mismatch(self):
+        self.organization = self.create_organization(slug="santry", region="us")
+
+        with pytest.raises(OrganizationMappingConsistencyException):
+            organization_mapping_service.upsert(
+                organization_id=self.organization.id,
+                update=RpcOrganizationMappingUpdate(
+                    slug="foobar", name="saaaaantry", region_name="us"
+                ),
+            )
+
+    def test_update_when_slug_matches_temporary_alias(self):
+        pass
+
+    def test_update_without_slug(self):
+        self.organization = self.create_organization(slug="santry", region="us")
+
+        organization_mapping_service.upsert(
+            organization_id=self.organization.id,
+            update=RpcOrganizationMappingUpdate(
+                slug=self.organization.slug,
```

Comment:
Which part of this is missing a slug?

Reply:
Deleting this, I was attempting to test the case where our org mapping upserts only included customer_id but the behavior has changed so now it always assumes defaults of empty string meaning this case isn't even possible anymore.